### DISCUSSION TITLE
What is RGB color space in the HEX_to_RGB function?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#1005] [Q&A] [answered]
Hi Colour Team,

In the HEX_to_RGB function (`colour.notation.HEX_to_RGB`), what is the RGB color space? Is it non-linear sRGB?
https://colour.readthedocs.io/en/develop/generated/colour.notation.HEX_to_RGB.html

Thanks

**Comment by thomasmansencal** [comment, marked as answer]
Hi @hminle,

So there is no RGB colourspace per-see, as it is mostly a change of notation. There are however some tiny smarts:

- `< 0` RGB colours are clamped
- `> 1` RGB  colours are normalised: They are first decoded with the sRGB EOTF, normalised to the maximum channel value, then re-encoded with the sRGB EOTF-1

Cheers,

Thomas

**Reply by hminle** [reply]
Hi @KelSolaar , so it is really depend on the input, right? For example, I open a color picker program on my sRGB display, I choose a hex color, then use the colour function to convert it to RGB colour space. The output will be non-linear sRGB.
Is that right?

**Reply by thomasmansencal** [reply]
It really depends on what the colour picker does under the hood, it is really hard to say though. The safe least common denominator is sRGB but if you were to pick a value in a paint program and that the file is colour managed, you would probably pick values in the RGB working space defined by the colour management system.

**Reply by hminle** [reply]
Oh, I got it, thank you.
